Cargo.toml

```diff
@@ -56,15 +56,13 @@ cosmic-client-toolkit = { git = "https://github.com/pop-os/cosmic-protocols//",
 # cosmic-settings-config = { git = "https://github.com/pop-os/cosmic-settings-daemon//", branch = "input_nobuild" }
 
 # For development and testing purposes
-# [patch.'https://github.com/pop-os/libcosmic']
-# libcosmic = { git = "https://github.com/pop-os/libcosmic//", branch = "font" }
-# cosmic-config = { git = "https://github.com/pop-os/libcosmic//", branch = "font" }
-# cosmic-theme = { git = "https://github.com/pop-os/libcosmic//", branch = "font" }
-# iced_futures = { git = "https://github.com/pop-os/libcosmic//", branch = "font" }
-
```

Comment:
This should be pointed at the branch. (Though it doesn't really matter since it would be changed before merging anyway.)